techlibs/xilinx/synth_xilinx.cc

```diff
@@ -49,10 +49,25 @@ struct SynthXilinxPass : public ScriptPass
 		log("    -top <module>\n");
 		log("        use the specified module as top module\n");
 		log("\n");
-		log("    -family {xcup|xcu|xc7|xc6v|xc5v|xc6s}\n");
+		log("    -family <family>\n");
 		log("        run synthesis for the specified Xilinx architecture\n");
 		log("        generate the synthesis netlist for the specified family.\n");
-		log("        default: xc7\n");
+		log("        supported values:\n");
+		log("        - xcup: Ultrascale Plus\n");
+		log("        - xcu: Ultrascale\n");
+		log("        - xc7: Series 7\n");
+		log("        - xc6s: Spartan 6\n");
+		log("        - xc6v: Virtex 6\n");
+		log("        - xc5v: Virtex 5 (EXPERIMENTAL)\n");
+		log("        - xc4v: Virtex 4 (EXPERIMENTAL)\n");
+		log("        - xc3sda: Spartan 3A DSP (EXPERIMENTAL)\n");
```

Comment:
Is there enough of a difference between the Spartan3 families to warrant a separate entry?

Reply:
I see from #1678 that Spartan 3/3E, Spartan 3A, Spartan 3A DSP all have different block RAM rules!

Reply:
Correct, and Spartan 3 vs Spartan 3E/3A also has different multiplier primitives (which we don't currently make use of beyond the basics that are identical, but we should do so in the future).